cms/templates/js/content-group-details.underscore

```diff
@@ -32,7 +32,7 @@
             </li>
         <% } else { %>
             <li class="action action-delete wrapper-delete-button" data-tooltip="<%- gettext('Cannot delete when in use by a unit') %>">
-                <button class="delete action-icon is-disabled" aria-disabled="true" disabled="disabled" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
+                <button class="delete action-icon is-disabled" disabled="true" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
```

Comment:
@clrux Yes, but I can't see the icon highlighted if I'm just using the keyboard.